app/views/root/_gem_base.html.erb

```diff
@@ -31,6 +31,12 @@
 %>
 
 <% content_for :head do %>
+  <script>
+    window.GOVUK = window.GOVUK || {};
+    window.GOVUK.analyticsVars = window.GOVUK.analyticsVars || {};
+    window.GOVUK.analyticsVars.gaProperty = '<%= Rails.application.config.ga_universal_id %>';
+    window.GOVUK.analyticsVars.gaPropertyCrossDomain = '<%= Rails.application.config.ga_secondary_id %>';
+  </script>
```

Comment:
It's probably worth seeking some other opinions on preferences of how these properties are set since it'll be on every page of GOV.UK

I somewhat wondered if we might want them to be meta elements `e.g <meta name="govuk:ga_universal_id" content="<%= Rails.application.config.ga_universal_id %>">` is perhaps a little cleaner and then get that data by JS, rather than mix things up. This would also mean we don't need to fix this later as an example of `unsafe-inline` JS, if we ever manage to get a content security policy implemented.

I feel like @andysellick is probably closer to this code than me though so be good to get his thoughts.

Reply:
Similar to Kevin, I'd prefer this to not be set as an inline script - a meta tag or a data attribute on the script itself would be more future proof.

Reply:
Great idea, SGTM! (you can probably tell I haven't done any frontend stuff professionally for over a decade 😅)

Stand by for updated implementation :)

Reply:
I agree with the suggestion although I think it'll be slightly more complicated to implement and we're looking to retire this analytics code in a year's time so if it turns out to be problematic then maybe don't worry about it too much?

Reply:
If you're ok with the inline script approach then happy to go with that 👍 (I haven't had time to redo it with `meta`, though I completely agree that would be the way to go if we weren't planning on deleting this soon-ish.)